README.md

```diff
@@ -1,6 +1,6 @@
 <div align="center">
   
-Intel® Neural Compressor
+Intel® Neural Compressor1
```

Comment:
does it make sense to have such number? can we have another naming?

Reply:
Sorry, it's changed to for test. Now it's restored.